src/Form.jsx

```diff
@@ -124,7 +124,8 @@ var Form = React.createClass({
         label = field.label,
         formfield = null,
         hasError = this.state.errorElements.indexOf(name) !== -1,
-        inputClass = hasError ? 'error' : '';
+        labelClass = field.labelClassname ? field.labelClassname : '',
```

Comment:
Is it because currently we don't append other classes to `inputClass` so we can safely set the default to `null` ? I'm thinking about the scenarios where we start doing `inputClass += ' some-classname'` like what we are doing `labelClass` later in this file.